crypto/x509/by_file.c

```diff
@@ -192,6 +192,9 @@ int X509_load_cert_crl_file(X509_LOOKUP *ctx, const char *file, int type)
     X509_INFO *itmp;
     BIO *in;
     int i, count = 0;
+
+    if (file == NULL)
+        return 0;
     if (type != X509_FILETYPE_PEM)
         return X509_load_cert_file(ctx, file, type);
     in = BIO_new_file(file, "r");
```

Comment:
A few lines blow there is a X509_STORE_add_cert/crl function where the return code is ignored,
could you please also take care of that?